swal-forms.js

```diff
@@ -160,13 +163,27 @@
       placeholder: field.placeholder || camelCaseToHuman(field.id),
       value: field.value || '',
       type: field.type || 'text',
+      options: field.options || [],
       isRadioOrCheckbox: function() {
         return isRadioOrCheckbox(input)
       },
       toHtml: function() {
-        return t("<input id='{id}' class='{clazz}' type='{type}' name='{name}'" +
-                  " value='{value}' title='{placeholder}' placeholder='{placeholder}'>" + 
-                "<label for='{name}'>{label}</label>", input)
+        if (input.type!=="select"){
+            var inputTag = t("<input id='{id}' class='{clazz}' type='{type}' name='{name}'" +
+                 " value='{value}' title='{placeholder}' placeholder='{placeholder}'>", input);
+        }else{
+            var inputTag = t("<select id='{id}' class='{clazz}' name='{name}'" +
+                 " value='{value}' title='{placeholder}' style='width:100%'>", input);
+            input.options.forEach(function(option){
```

Comment:
I like the overall approach to add <select> capabilities, though this particular call could be replaced buy a `reduce` call (since we are basically generating a single string by transforming a set of items).